chain/client/src/types.rs

```diff
@@ -155,6 +156,19 @@ impl Message for GetBlock {
     type Result = Result<BlockView, String>;
 }
 
+/// Get block with the block merkle tree. Used for testing
+pub struct GetBlockWithMerkleTree(pub BlockIdOrFinality);
```

Comment:
This type seems to exist only to workaround the actix framework. Can we rename it to something like: `RequestForBlockWithMerkleTree`?

Reply:
I would prefer it to be consistent with what we already have, i.e, `GetBlock`.